src/Stoa.ts

```diff
@@ -2081,9 +2082,8 @@ class Stoa extends WebService {
                     if (changes)
                         logger.info(
                             `Saved a pre-image utxo : ${pre_image.utxo.toString().substr(0, 18)}, ` +
-                                `hash : ${pre_image.hash.toString().substr(0, 18)}, pre-image height : ${
-                                    pre_image.height
-                                }`,
+                            `hash : ${pre_image.hash.toString().substr(0, 18)}, pre-image height : ${pre_image.height
```

Comment:
Is there a reason why you use `substr`?
Why don't we loged the entire hash?

Reply:
This was used before, in fact entire hash can be logged

Reply:
Entire hash is now being logged